WordPress/openverse
# Deploy staging thumbnails along with staging API

## Fixes



Related to https://github.com/WordPress/openverse-infrastructure/pull/544

## Description


Deploys the staging thumbnails service after the API deploys. While we could technically deploy them in parallel, it would just be more complicated, especially for something that is still just to test a theory. If we stick with this in the future we can look into parallel service deployments if needed.

## Testing Instructions

I copy/pasted the new step in the `deploy-api` job for the regular API deployment and updated the step name and dispatch event name to match the event name in the deployment workflow file: https://github.com/WordPress/openverse-infrastructure/pull/544

Confirm that I haven't left anything else that needs changing there.

I've also updated the CI/CD workflow docs, so please read those changes and confirm they are accurate.

## Checklist


- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I tried running the project locally and verified that there are no visible errors.
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
